calendar/api/parent/ext-calendar-provider.js

```diff
@@ -103,11 +103,52 @@ class ExtCalendarProvider extends cal.provider.BaseClass {
           : ["unsubscribe"];
       case "requiresNetwork":
         return !(this.capabilities.requires_network === false);
+      case "itip.transport":
+        if (this.extension.emitter.has("calendar.provider.onSend")) {
+          return this;
+        }
+        break;
     }
 
     return super.getProperty(name);
   }
 
+  scheme = "mailto";
+
+  async sendItems(aRecipients, aItipItem) {
+    let { jsmime } = ChromeUtils.import("resource:///modules/jsmime.jsm");
```

Comment:
Import at the top of the file, not on every invokation. JSMime is going to be loaded at TB start anyway, no point in lazy loading. And, FWIW, lazy load slows down every function call a little bit, even if it's cached.